test/test_installation.py

```diff
@@ -114,16 +111,15 @@ def _create_match_list(self) -> List[str]:
             "share/doc",
             "share/doc/unified-memory-framework",
         ]
-        if self.examples:
-            examples_dir = Path(self.workspace_dir, "examples")
-            examples_dirs = [dir for dir in examples_dir.iterdir() if dir.is_dir()]
-            examples = [
-                f"share/doc/unified-memory-framework/examples/{file_path.name}"
-                for example_dir in examples_dirs
-                for file_path in example_dir.iterdir()
-            ]
-            examples.insert(0, "share/doc/unified-memory-framework/examples")
-            share.extend(examples)
+        examples_dir = Path(self.workspace_dir, "examples")
+        examples_dirs = [dir for dir in examples_dir.iterdir() if dir.is_dir()]
+        examples = [
+            f"share/doc/unified-memory-framework/examples/{file_path.name}"
+            for example_dir in examples_dirs
+            for file_path in example_dir.iterdir()
+        ]
+        examples.insert(0, "share/doc/unified-memory-framework/examples")
```

Comment:
could we just use share/doc/umf/examples path? this would be consistent with /include/umf/..

Reply:
same with lib/cmake/umf - I know this is not strictly related to this PR but it would be good to cleanup this. Please let me know if you could update this also or we'll do this in separate PR

Reply:
Created #370 for this

Reply:
ok @PatKamin thanks!